### ISSUE TITLE
How to use LR schedulers?

### ISSUE THREAD

**Issue by brianhhu** [pytorch/xla#797]
Can you provide an example of how to use torch.optim LR schedulers with Pytorch/XLA? I've been basing my code on the [test examples](https://github.com/pytorch/xla/tree/master/test), but not sure where to define the LR scheduler and where to step the scheduler since it looks like the optimizer is re-initialized each training loop. Thanks!

**Comment by dlibenzi** [comment]
Good point!
Let me gather some thoughts and will get back to you.

**Comment by dlibenzi** [comment]
You can try to patch in https://github.com/pytorch/xla/pull/800 to see if that works for you.

**Comment by brianhhu** [comment]
Ok, thanks- I still need to give this a try. Is there an easy way to apply a patch if I'm using a prebuilt compute VM image and not building from source?

**Comment by dlibenzi** [comment]
I have just merged the PR. It should be showing up in tomorrow's nightly VMs and Dockers.
